Std/Data/Array/Lemmas.lean

```diff
@@ -844,3 +844,61 @@ instance [DecidableEq α] (a : α) (as : Array α) : Decidable (a ∈ as) :=
 /-! ### erase -/
 
 @[simp] proof_wanted erase_data [BEq α] {l : Array α} {a : α} : (l.erase a).data = l.data.erase a
+
+/-! ### swap -/
+
+@[simp] theorem get_swap_right (a : Array α) {i j : Fin a.size} :
+    (a.swap i j)[j.val]'(a.size_swap .. |>.symm ▸ j.isLt) = a[i] :=
```

Comment:
```suggestion
    (a.swap i j)[j.val] = a[i] :=
```

Reply:
Unneeded.

Reply:
oh this, sorry this is an artifact of having additionally tested against v4.6.1, forgot to remove it.